_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+
+### Prediction
+This answers the question of $$ P(Y | X) $$ which means the probability of Y if X happens to take a particular value. This can be found out just from the observational data. We don't need any graphs, or causal realtions to find this value. 
+
+### Intervention
+This answers the question of $$ P(Y | do(X)) $$ which means what is the probability of Y if X is set to a particular value keeping all the other variables same. For this, we definitely need to know all the causal relations of all the variables.
+
+### Counterfactual Thinking
+This answers the question of $$ P(y_x | x', y') $$, which is the probability that event Y = y would be observed had X been x, given that we actually observed X to be $$ x' $$ and and Y to be $$ y' $$. For this, again we require causal relations via the structural equation model.
+
+## Identification of Causal Relation
+
+### Causal DAGs
+Each edge in a causal DAG corresponds to direct causal influence between the nodes and it is asymmetric.
+Let $$ P_x(V) $$ be the distribution resulting from intervention do(X=x). Then a DAG is a causal DAG if
+* $$ P_x(V) $$ is Markov relative to G. This distribution is still factorizable according to the Markov property (product of proabbility of each node given its parents) with respect to G
+    
+* $$ P_x(V_i = v_i) = 1 $$ for $$ V_i \in X $$ and $$ v_i $$ consistent with X=x. This is so because we have performed intervention on X and we have set that to x. Hence we know those variables with absolute certainty.
+    
+* $$ P_x(V_i | PA_i) = P(V_i | PA_i) $$ for all $$ V_i \notin X $$ where $$ PA_i $$ represents the parents of $V_i$. That means the conditional distributions of all the nodes not included in the intervention remains the same before and after the intervention. We can see this graphically since while doing intervention, we just delete all the incoming edges into the nodes on which intervention is performed. So the other nodes remain unchanges(in terms of parents)
+
+### Randomized Control Experiments
+We change a controlled variable and observe the change in an outcome variable. All the other variables that influence the outcome variable is either fixed or varied at random. So now any change observed in the outcome variable must have been due to the change in the controlled variable.
+
+Its disadvantage is that its expensive and in many cases impossible to do. The outcome variable can depend on many other variables. So, to find one causal relation, we would need a lot of data and we need to ensure that the other variables are fixed or varied randomly across the experiments which is a very hard task.
+
+### Kidney Stone Experiment
+Lets revisit the kidney stone treatment example. Let R = Recovery, T = Treatment and S = Stone size, then
+
+$$ P(R|T) = \sum_S P(R|S,T)P(S|T) $$
+$$ P(R|do(T)) = \sum_S P(R|S,T)P(S) $$
```

Comment:
This equation doesn't render, please fix.

Changing "|" to "\mid" in the LaTeX code seems to fix the rendering problem elsewhere, so try that here.